airflow/config_templates/default_airflow.cfg

```diff
@@ -317,6 +323,12 @@ celery_config_options = airflow.config_templates.default_celery.DEFAULT_CELERY_C
 # ETA you’re planning to use. Especially important in case of using Redis or SQS
 visibility_timeout = 21600
 
+# In case of using SSL
```

Comment:
Why keep this in uppercase? Why keep it prepended with `celery_`?

Reply:
Good point Bolke, I've removed the `celery_` prefixes.

Reply:
Still uppercase? Please use lowercase as we do everwhere else and celery4 moved to lowercase.

Reply:
Good point @bolkedebruin, I've lowercased the keys